### ISSUE TITLE
Unable to pass Id in React Multi Email

### ISSUE THREAD

**Issue by Prasanna-mediwave** [axisj/react-multi-email#88]
working in accessibility I'm unable to pass ID to the input field

**Comment by abdullahguersu** [comment]
I think this is duplicate of #87
